packages/langium-cli/src/generate.ts

```diff
@@ -141,12 +141,17 @@ export async function generate(config: LangiumConfig, options: GenerateOptions):
         const buildResult = all.get(absGrammarPath);
         if (buildResult) {
             const grammar = buildResult.document.parseResult.value as Grammar;
+            if(!grammar.isDeclared) {
+                log('error', options, `The grammar top level object is missing in ${absGrammarPath}`.red);
```

Comment:
Proposal: change the message to "${absGrammarPath}: The entry grammar must start with the 'grammar' keyword."